src/routes/embedded/support/unpartitioned-state/unpartitioned-state.view.tsx

```diff
@@ -137,14 +161,13 @@ export function UnpartitionedStateMissingEmbeddedView() {
   }, [unpartitionedStateStatus]);
 
   const needsConfirmation = !unpartitionedStateConfirmed && authStatus === "noAuth";
-  const requestAccessButtonText =
-    unpartitionedStateStatus === "rejected" || errorsWhileRequestingAccess === 0 ? "Request access" : "Try again";
+  const requestAccessButtonText = errorsWhileRequestingAccess === 0 ? "Check access" : "Check access again";
+  const accessTo = browserInfo.isBrave ? "embedded content" : "third-party cookies";
```

Comment:
Replaced the previous wording with more accurate one, as most users that end up in this view are aware of these privacy-related terms. Also, it's confusing to mention made-up terms like "cross site wallet syncing" and then show screenshots below that mention none of that.